portugues/titanic/arrumar/arrumar.md

```diff
@@ -0,0 +1,3 @@
+Você mesmo com todos os os programadores do barco percebem que não tem mais jeito, e colidem com toda a forca no bugberg. O aplicativo nunca mais abre.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.